helm_deploy/hmpps-incentives-api/values.yaml

```diff
@@ -12,6 +12,9 @@ generic-service:
     tag: app_version    # override at deployment time
     port: 8080
 
+  retryDlqCronjob:
+    enabled: true
```

Comment:
This setting creates a CronJob very similar (aside from the `schedule`) to the one we had before: https://github.com/ministryofjustice/hmpps-helm-charts/blob/85b24e1789449cf3306fe9229cf6dbef088d9ae2/charts/generic-service/templates/retry-dlq-cronjob.yaml

Reply:
When the scheduled downtime is enabled (as it is for `dev` and `preprod` in our case) this is the schedule used for this CronJob:

```
# every 10 minutes between 7am and 10pm UTC Monday-Friday
retryDlqSchedule: "*/10 7-21 * * 1-5"
```

https://github.com/ministryofjustice/hmpps-helm-charts/blob/85b24e1789449cf3306fe9229cf6dbef088d9ae2/charts/generic-service/values.yaml#L237